crafters/nlp/PDFExtractorSegmenter/tests/test_pdf_flow.py

```diff
@@ -14,7 +14,7 @@ def test_pdf_flow():
 
     f = Flow().add(uses='PDFExtractorSegmenter')
     with f:
-        f.index_files([path] )
+        f.index_files([path])
 
     with f:
```

Comment:
it does not make sense to `index` and `search` in this example since both calls would do the same, pick one and do some validation as `output_fn`